### PR TITLE
API changes for Push Notifications (CU-10xfkhr)

### PR DESCRIPTION
**Pull request by trisapeace**
- Just needed a way to retrieve the ActiveAlerts from the DB and give them to brave-alert-lib.

## Test plan
- :heavy_check_mark: Passes all the tests in https://github.com/bravetechnologycoop/brave-alert-lib/pull/27

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-10xfkhr Backend & Frontend: Alert push notifications sending and handling](https://app.clickup.com/t/10xfkhr)

File: server/db/db.js [hunk 1]
```diff
@@ -663,17 +663,37 @@ async function getInstallationWithSessionId(sessionId, clientParam) {
...
 
-    // Historic Alerts are those with status "Completed" or that were last updated longer ago than the SESSION_RESET_TIMEOUT
-    const { rows } = await client.query(
+    return results.rows
+  } catch (err) {
+    helpers.logError(err.toString())
+  }
+}
+
+async function getHistoricAlertsByAlertApiKey(alertApiKey, maxHistoricAlerts, maxTimeAgoInMillis, clientParam) {
```
Comment (trisapeace) [inline server/db/db.js:693]: While I was in the area, changed this one to use `helpers.runQuery` as well.

File: server/package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "dependencies": {
     "body-parser": "^1.18.3",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-add-onesignal",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-add-onesignal-part-2",
```
Comment (trisapeace) [inline server/package.json:8]: Reminder to change this once brave-alert-lib is merged.

Comment (trisapeace) [reply server/package.json]: Done.

File: server/db/db.js [hunk 3]
```diff
@@ -663,17 +663,37 @@ async function getInstallationWithSessionId(sessionId, clientParam) {
...
+    const results = await helpers.runQuery(
+      'getActiveAlertsByAlertApiKey',
+      `
+      SELECT s.id, s.state, b.unit, s.num_presses, i.incident_categories, s.created_at
+      FROM sessions AS s
+      LEFT JOIN buttons AS b ON s.button_id = b.button_id
+      LEFT JOIN installations AS i ON s.installation_id = i.id
+      WHERE i.alert_api_key = $1
+      AND (
+        s.state <> $2
```
Comment (schwarrrtz) [inline server/db/db.js:677]: the `<>` operator was confusing to me at first. I had to look it up - personally I think that `!=` would make the code easier to maintain, but I don't feel like it needs to be changed.

File: server/db/db.js (hunk 1 above)
Comment (schwarrrtz) [reply server/db/db.js:693]: nice, thanks!

**Review by schwarrrtz — APPROVED** [review]
this looks great. I like the thorough test cases!

File: server/db/db.js (hunk 3 above)
Comment (trisapeace) [reply server/db/db.js]: You're right. I changed it. Thanks!

**Pull request merged**